### DISCUSSION TITLE
Possible switching CANCEL and SET on QPopupEdit

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#12259] [General - Components / Directives / etc] [answered]
Is it possible to switch the standard buttons CANCEL and SET from position on a QPopupEdit:

![image](https://user-images.githubusercontent.com/19785727/151692116-a4dfca0b-7683-4856-b08d-5d238c04a4ab.png)

**Comment by MilosPaunovic** [comment, marked as answer]
You can use `default` slot and override buttons the way you need 'em.

**Reply by eugenevk** [reply]
Works like a charm. Thanks @MilosPaunovic 

![image](https://user-images.githubusercontent.com/19785727/151792148-093b211d-c302-4d45-8e55-b67fc488066b.png)
